app/core/ai_monitoring.py

```
@@ -0,0 +1,186 @@
+from typing import Dict, List, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)